examples/getting-started/MAC_with_podman/README.md

```diff
@@ -0,0 +1,80 @@
+# Getting Started on a MAC M-Series (arm64-based)
+
+- Install 'brew'
```

Comment:
```suggestion
- Install 'brew'.
```

Reply:
I think this should be either _Install Homebrew._ or _Install `brew`._

https://brew.sh/